File: javascript/ex028.js
console.clear();


const { randomInt } = require('crypto');

const computador = randomInt(0, 6);
const jogador = 3;

console.log("PROCESSANDO...");

setTimeout(() => {}, 400);

if (jogador === computador) {
    console.log("Parabens, voce conseguiu me vencer");
} else {
    console.log(`Ganhei, eu pensei no numero ${computador} e não no ${jogador}`);
}

File: python3/ex028.py
from random import randint
from time import sleep


computador = randint(0, 5)
jogador = int(input("Em que nome eu persei? "))

print("PROCESSANDO")
sleep(0.4)


if jogador == computador:
    print("Parabens, voce conseguiu me vencer")

else:
    print(F"Ganhei, eu pensei  no numero {computador} e náo no {jogador}")